tests/utility/hash_test.cu

```diff
@@ -48,6 +48,63 @@ __host__ __device__ bool check_hash_result(typename Hash::argument_type const& k
   return (h(key) == expected);
 }
 
+// Overload for hash functions without a seed
+template <typename Hash>
+__host__ __device__ bool check_hash_result(typename Hash::argument_type const& key,
+                                           typename Hash::result_type expected) noexcept
+{
+  Hash h;
+  return (h(key) == expected);
+}
+
+template <typename OutputIter>
+__global__ void check_identity_hash_result_kernel(OutputIter result)
+{
+  int i = 0;
+
+  result[i++] = check_hash_result<cuco::identityhash<signed char>>(SCHAR_MIN, SCHAR_MIN);
+  result[i++] = check_hash_result<cuco::identityhash<signed char>>(SCHAR_MAX, SCHAR_MAX);
+
+  result[i++] = check_hash_result<cuco::identityhash<int32_t>>(INT32_MIN, INT32_MIN);
```

Comment:
Let's use [std::numeric_limits](https://en.cppreference.com/w/cpp/types/numeric_limits) instead of C macros. In modern C++ code, we want to get rid of any C expressions when possible, e.g. using STL algorithms instead of raw for/while loops, and here `std::numeric_limits<int32_t>::min()` instead of INT32_MIN.

Reply:
Sounds good to me!